apps/ios_rpc/tvmrpc/TVMRuntime.mm

```diff
@@ -45,7 +45,8 @@
   throw tvm::runtime::InternalError(file, lineno, message);
 }
 
-void LogMessageImpl(const std::string& file, int lineno, const std::string& message) {
+void LogMessageImpl(const std::string& file, int lineno, int level, const std::string& message) {
+  // TODO: Output logging level
```

Comment:
remove?

Reply:
Since the iOS build is currently skipped in CI, I'll remove this TODO and address it later.